src/IBM.Watson.SpeechToText.v1/Test/Integration/SpeechToTextServiceIntegrationTest.cs

```diff
@@ -587,6 +589,65 @@ public void TestRecognize_Success()
             Assert.IsNotNull(recognizeResult.Result);
             Assert.IsNotNull(recognizeResult.Result.Results);
             Assert.IsTrue(recognizeResult.Result.Results.Count > 1);
+        }
+
+        [TestMethod]
+        public void TestRecognizeUsingWebsockets()
+        {
+            service.WithHeader("X-Watson-Test", "1");
+            RecognizeCallback callback = new RecognizeCallback();
+
+            string Name = @"SpeechToTextTestData/sample1.wav";
+
+            try
+            {
+                byte[] filebytes = File.ReadAllBytes(Name);
+                MemoryStream stream = new MemoryStream(filebytes);
+
+                callback.OnOpen = () =>
+                {
+                    System.Diagnostics.Debug.WriteLine("On Open");
+                };
+                callback.OnClose = () =>
+                {
+                    System.Diagnostics.Debug.WriteLine("On Close");
+                };
+                callback.OnMessage = (speechResults) =>
+                {
+                    System.Diagnostics.Debug.WriteLine("On Message");
+                    Assert.IsNotNull(speechResults);
+                };
+                callback.OnError = (err) =>
+                {
+                    System.Diagnostics.Debug.WriteLine("On error");
+                    System.Diagnostics.Debug.WriteLine(err);
+                };
+                service.RecognizeUsingWebSocket(
+                    callback: callback,
+                    audio: stream,
+                    contentType: RecognizeEnums.ContentTypeValue.AUDIO_WAV,
+                    interimResults: true,
+                    model: RecognizeEnums.ModelValue.EN_US_BROADBANDMODEL,
+                    inactivityTimeout: 55,
+                    keywords: new List<string>{"could"},
+                    keywordsThreshold: 0.8f,
+                    timestamps: true,
+                    profanityFilter: true,
+                    smartFormatting: true,
+                    speakerLabels: true,
+                    processingMetrics: true,
+                    processingMetricsInterval: 0.9f,
+                    audioMetrics: true,
+                    endOfPhraseSilenceTime: 0,
+                    splitTranscriptAtPhraseEnd: true,
+                    speechDetectorSensitivity: 1.0f,
+                    backgroundAudioSuppression: 0.5f
+                    );
+            }
```

Comment:
Are you asserting anything in this test?

Reply:
just checking that the `speechResults` in `OnMessage` is not null